website/.eslintrc.json

```
@@ -0,0 +1,3 @@
+{
+  "extends": "next/core-web-vitals"
```

Try to fix CI error:
```suggestion
  "root": true,
  "extends": "next/core-web-vitals"
```